### PR TITLE
Sprint 30

### PR DESCRIPTION
**Pull request by anselmbradford**

### CODE REVIEW

File: _includes/macros/activity-snippets.html [hunk 1]
```diff
@@ -40,63 +48,66 @@
 
    Description:
 
-   Private method to list titles of recent document type posts when given:
+   List titles of recent document type posts when given:
+
+   doctype:           The document type you wish to display.
+                      Note that newsroom is not a supported document tpye in this
```
Comment (anselmbradford) [inline _includes/macros/activity-snippets.html]: Typo: `tpye`

File: _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
+<section class="block
```
Comment (anselmbradford) [inline _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Drop `-form` from filename to shorten it slightly, it's in the `/forms/` subdirectory.

File: _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                <div class="custom-select">
+                    <select class="custom-select_select"
+                            data-placeholder="Select state"
+                            id="organization-state"
+                            required>
+                        <option value="">Select state</option>
+                        <option value="AL">Alabama</option>
+                        <option value="AK">Alaska</option>
+                        <option value="AZ">Arizona</option>
+                        <option value="Other">Other States to be added</option>
```
Comment (anselmbradford) [inline _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Add `TODO:` so we can find this with search.

File: _includes/templates/forms/file-a-privacy-complaint-form.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+                <div class="custom-select">
+                    <select class="custom-select_select"
+                            data-placeholder="Select State"
+                            id="organization-state"
+                            required>
+                        <option value="">Select state</option>
+                        <option value="AL">Alabama</option>
+                        <option value="AK">Alaska</option>
+                        <option value="AZ">Arizona</option>
+                        <option value="Other">Other States to be added</option>
```
Comment (anselmbradford) [inline _includes/templates/forms/file-a-privacy-complaint-form.html]: Add `TODO:` so this shows up in search.

File: sub-pages/_single.html [hunk 5]
```diff
@@ -85,20 +84,43 @@ <h2 class="h3">
...
-        <section class="office_activities">
-            {% include "templates/activities-feed.html" %}
-        </section>
-    </aside>
+    <section class="office_activities
+                    block
+                    block__flush-sides
+                    block__bg
+                    u-mb0
+                    u-mt0">
```
Comment (anselmbradford) [inline sub-pages/_single.html]: Replace `u-mb0` and `u-mt0` with `block__flush-top` and `block__flush-bottom`.

**Comment by anselmbradford** [comment]
Review done for Sprint 30. I don't see anything major.

**Comment by KimberlyMunoz** [comment]
One quick thing: I forgot to update the changelog for my PR.

https://github.com/cfpb/cfgov-refresh/pull/693

File: sub-pages/_single.html [hunk 6]
```diff
@@ -85,20 +84,43 @@ <h2 class="h3">
                     block
                     block__padded-top
                     block__border-top">
-        {{ sub_page.body_content }}
+        {% set content = { 'markup' : sub_page.body_content } %}
+        {% for form_name in vars.forms %}
```
Comment (sebworks) [inline sub-pages/_single.html]: This isn't a great solution. Need to rethink how we handle the forms and the form placeholders coming from WP.

File: _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html (hunk 2 above)
Comment (KimberlyMunoz) [inline _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: If we change the file names, we might have to have Sarah go back into the CMS and change the name of the forms that the template is looking for.

File: _includes/related-links.html [hunk 7]
```diff
@@ -64,8 +64,12 @@ <h1 class="header-slug">
    ========================================================================== #}
 
 {%- macro link_template(link) %}
+{% if link[0] == '' %}
+<a class="list_link jump-link jump-link__right list_link__disabled">
```
Comment (jimmynotjim) [inline _includes/related-links.html]: These classes should be stacked

File: _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+            </label>
+            <div class="custom-select">
+                <select class="custom-select_select"
+                        data-placeholder="Select Time"
+                        id="contact-time">
+                      <option value="">Select time</option>
+                      <option>8:00 a.m</option>
+                 </select>
+                 <span class="custom-select_icon
+                                 cf-icon
```
Comment (jimmynotjim) [inline _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: left align class names

File: _includes/macros/activity-snippets.html (hunk 1 above)
Comment (anselmbradford) [inline _includes/macros/activity-snippets.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/694

File: _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html (hunk 2 above)
Comment (anselmbradford) [inline _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Ahh and it probably makes sense on the CMS-side to have form in the name I'm guessing. cc @kurtw

File: _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html (hunk 3 above)
Comment (anselmbradford) [inline _includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/695

File: _includes/templates/forms/file-a-privacy-complaint-form.html (hunk 4 above)
Comment (anselmbradford) [inline _includes/templates/forms/file-a-privacy-complaint-form.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/695

File: sub-pages/_single.html (hunk 5 above)
Comment (anselmbradford) [inline sub-pages/_single.html]: https://github.com/cfpb/cfgov-refresh/issues/696

**Pull request merged**